Examples/java/build.gradle.kts

```
@@ -0,0 +1,107 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: CC-BY-SA-4.0
+import java.io.File
+import java.io.FileInputStream
+import java.util.Properties
+plugins {
+    `java`
+    `java-library`
+}
+var props = Properties().apply {
+    load(FileInputStream(File(rootProject.rootDir, "../../project.properties")))
+}
+var dafnyVersion = props.getProperty("dafnyVersion")
+group = "software.amazon.cryptography"
+var mplVersion = props.getProperty("mplVersion")
+description = "AWS Cryptographic Material Providers Library Examples"
+var slf4jVersion = "1.7.32"
+var esdkVersion = "3.0.1"
+dependencies {
+    // The Library being demonstrated
+    implementation("software.amazon.cryptography:aws-cryptographic-material-providers:${mplVersion}")
+    // Dafny dependencies
+    implementation("org.dafny:DafnyRuntime:${dafnyVersion}")
+    implementation("software.amazon.smithy.dafny:conversion:0.1.1")    
+    // sdk dependencies
+    implementation(platform("software.amazon.awssdk:bom:2.26.3"))
+    // ESDK uses the MPL
```

Is this only used in the downstream testing? Maybe
```suggestion
    // Used in downstream testing
```